File: app/src/main/java/com/karigor/friends/model/Result.java
package com.karigor.friends.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Result implements Serializable {

    @SerializedName("gender")
    @Expose
    private String gender;

    @SerializedName("name")
    @Expose
    private Name name;

    @SerializedName("picture")
    @Expose
    private Picture picture;

    @SerializedName("phone")
    @Expose
    private String phone;

    @SerializedName("email")
    @Expose
    private String email;

    @SerializedName("location")
    @Expose
    private Location location;


    public String getGender() {
        return gender;
    }

    public Name getName() {
        return name;
    }

    public Picture getPicture() {
        return picture;
    }

    public String getPhone() {
        return phone;
    }

    public String getEmail() {
        return email;
    }

    public Location getLocation() {
        return location;
    }
}

File: app/src/main/java/com/karigor/friends/model/Result.kt
package com.karigor.friends.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import java.io.Serializable

class Result : Serializable {
    @SerializedName("gender")
    @Expose
    val gender: String? = null

    @SerializedName("name")
    @Expose
    val name: Name? = null

    @SerializedName("picture")
    @Expose
    val picture: Picture? = null

    @SerializedName("phone")
    @Expose
    val phone: String? = null

    @SerializedName("email")
    @Expose
    val email: String? = null

    @SerializedName("location")
    @Expose
    val location: Location? = null
}